pkg/ui/workspaces/cluster-ui/src/databasesV2/constants.ts

```diff
@@ -11,7 +11,7 @@
 export enum DatabaseColName {
   NAME = "Name",
   SIZE = "Size",
-  RANGE_COUNT = "Range Count",
   TABLE_COUNT = "Table Count",
   NODE_REGIONS = "Regions / Nodes",
+  SCHEMA_INSIGHTS = "Schema Insights",
```

Comment:
why is this diff here if schema insights is being removed below?

Reply:
Was just here for completeness at the time of mocking up to match designs. I think we're still trying to decide what to do with it so we can remove it later.

Reply:
@dhartunian Went ahead and removed it for now.